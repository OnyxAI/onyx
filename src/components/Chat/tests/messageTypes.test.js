import { messageTypes } from '../messageTypes';

describe('Chat messageTypes', () => {
  it('Should render snapshot', () => {
    expect(messageTypes).toMatchSnapshot();
  });
});
