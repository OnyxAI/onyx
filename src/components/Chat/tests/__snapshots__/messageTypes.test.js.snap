// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chat messageTypes Should render snapshot 1`] = `undefined`;
